contrib/cmake/CMakeLists.txt

```diff
@@ -405,12 +405,14 @@ endif()
 if( ${WITH_IECORE_USD} )
     find_package( USD REQUIRED )
     include_directories( ${CORTEX_SOURCE_DIR}/contrib/IECoreUSD/include )
+    include_directories( ${USD_INCLUDE_DIR} )
+    link_directories( ${USD_LIBRARY_DIR} )
     # Library
     file( GLOB IECOREUSD_CXX_FILES  ${CORTEX_SOURCE_DIR}/contrib/IECoreUSD/src/IECoreUSD/*.cpp )
     file( GLOB IECOREUSD_H_FILES  ${CORTEX_SOURCE_DIR}/contrib/IECoreUSD/include/IECoreUSD/*.h ${CORTEX_SOURCE_DIR}/contrib/IECoreUSD/include/IECoreUSD/*.inl )
     file( GLOB IECOREUSD_PY_FILES  ${CORTEX_SOURCE_DIR}/contrib/IECoreUSD/python/IECoreUSD/*.py )
     add_library( IECoreUSD SHARED ${IECOREUSD_CXX_FILES} ${IECOREUSD_H_FILES} )
-    target_link_libraries( IECoreUSD IECore IECoreScene ${BASE_LIBS} )
+    target_link_libraries( IECoreUSD IECore IECoreScene ${BASE_LIBS} ${USD_LIBRARIES} usdGeom )
```

Comment:
Why issn't `usdGeom` included in the  `USD_LIBRARIES` variable?

Reply:
My choice is that any schema-generated lib would be added separately. Possibly we could have ${USD_SCHEMA_LIBRARIES} and bundle all of them in there, like any libs found with the usd prefix on them.